community-membership.md

```diff
@@ -93,6 +93,27 @@ code reviews and work towards becoming an *approver* for the subproject that
 they are active in.  Acceptance of code contributions requires at least one
 approver in addition to the reviews by *members.*
 
+## Triager
+
+Triagers assist the maintainers and approvers with project management and 
+backlog organization. The specific workflows and triage requirements depend on 
+the project, and are set by the project maintainers.
+
+Defined by: [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)
+
+Triagers are not required to make code contributions, but are encouraged to be
```

Comment:
We have Members who today make code contributions. Triager is a more permissive role and I would want to grant this role to several active Members who work on Collector. Do we need this wording here? It may be a source of confusion and can be seen as a role that is different from Member, but it really is a step up from Member if I understand it correctly (at least in terms of formal Github permissions).

Reply:
Thanks, I updated the wording to clarify this more. I do think we need to mention that code contributions are not *required* for this role, as that is rather unique.